isotope-intensity-estimation.Rmd

```diff
@@ -153,19 +155,22 @@ cmps <- cmps[-which(is.na(cmps$exactmass)), ]
 CHNOPS <- c("C", "H", "N", "O", "P", "S", "Cl")
 counts0 <- lapply(cmps$formula, function(frml) {
   res <- countElements(frml)
-  row <- data.frame(matrix(0, nrow = 1, ncol = length(CHNOPS) + 1))
+  row <- numeric(length(CHNOPS) + 1)
```

Comment:
I used a `numeric` instead of a `data.frame` because that should be faster (compared to `rbind,data.frame`).